spec/amber/router/pipe/powered_by_amber_spec.cr

```diff
@@ -0,0 +1,29 @@
+require "../../../../spec_helper"
+
+module Amber
+  module Pipe
+    describe PoweredByAmber do
+      context "Adds X-Powered-By: Amber to response" do
+        pipeline = Pipeline.new
+
+        pipeline.build :web do
+          plug PoweredByAmber.new
+        end
+
+        Amber::Server.router.draw :web do
+          options "/test", HelloController, :world
```

Comment:
Travis is failing:

```
Route: OPTIONS /test is duplicated. (Amber::Exceptions::DuplicateRouteError)
```

`/test` is already in use. Please use another path here, something like `/poweredbyamber`